sdk/state/state_test.go

```diff
@@ -391,6 +414,177 @@ func Test(t *testing.T) {
 	assert.EqualValues(t, initiatorEscrowResponse.Balances[0].Balance, fmt.Sprintf("%.7f", float64(iBalanceCheck)/float64(1_000_0000)))
 }
 
+func TestOpenUpdatesCoordinatedClose(t *testing.T) {
```

Comment:
@acharb this test seems very long which makes it hard to follow, and much much harder to maintain.

not a blocker, but if we can break it up in a future PR soon then that would make it a little easier to follow/maintain. It may be a little more complicated to do and we may need to invest in developing a good test framework, but something we should discuss with @leighmcculloch once he's back.

Reply:
I agree. This state_test.go can be cleaned up. I couldn't figure out a good way to write this second test w/o duplicating a lot of the code in the original test. 

I think once the majority of the sdf/state code is written this integration test can be broken up / looked at holistically and made more modular so there's less dup code. I didn't think now is the right time to do that though since stuff is changing frequently because early in the project

Reply:
> w/o duplicating a lot of the code

We don't need to keep our tests DRY. Let's keep tests isolated from each other. It makes it easier to delete them when we don't need them, or for them to evolve independently of each other.

Also, this is likely to get better over time because the more functionality we add to the SDK, the tests will get naturally smaller as functionality moves out of these integration tests into the SDK.